### ISSUE TITLE
[Bug] LSPosed app disappeared and I can't access it.

### ISSUE THREAD

**Issue by Asmageddon** [LSPosed/LSPosed#2738]
### Steps to reproduce/复现步骤

No idea.

### Expected behaviour/预期行为

LSPosed shortcut exists.

### Actual behaviour/实际行为

LSPosed has no app/shortcut/notification to the manager, but installed xposed modules work.

### Xposed Module List/Xposed 模块列表

```shell
XPrivacyLua
```

### Magisk Module List/Magisk 模块列表

```shell
Zygisk - LSPosed
Shamiko
```

### LSPosed version/LSPosed 版本

6990

### Android version/Android 版本

11

### Magisk version/Magisk 版本

26.1

### Riru version/Riru 版本

N/A

### Version requirement/版本要求

- [X] I am using latest debug CI version of LSPosed and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志

Cannot access manager.

**Comment by Asmageddon** [comment]
@yujincheng08 What am I supposed to do about this, then?

**Comment by Iscle** [comment]
Same here! Clean Android install, no Xposed Manager to be seen anywhere!

**Comment by Asmageddon** [comment]
@Iscle Some issues mentioned a secret dialer code, apparently if you type `*#*#5776733#*#*` into dialer, it will open the LSPosed manager for you.

**Comment by Iscle** [comment]
> @Iscle Some issues mentioned a secret dialer code, apparently if you type `*#*#5776733#*#*` into dialer, it will open the LSPosed manager for you.

I ended up downgrading to the previous version

**Comment by ddxv** [comment]
> I fixed it by extracting the APK from the zip and installed the manager manually
I also did this, after installing LSPosed in Magisk the first time, restarting the emulator twice and enable/disabling LSPOsed, I still didn't see the LSPosed app on the home screen or in the app drawer.

I installed a FileManager and extracted the `manager.apk` from the LSPosed release Zip folder and installed it. This appears to have worked as I now have a LSPosed app on the home screen.
